kcp/pkg/deploy/assets/sample-chart/templates/zz-variables.yaml

```diff
@@ -0,0 +1,9 @@
+{{- $cn := printf "%s-webhook.%s.svc" .Release.Name .Release.Namespace }}
+{{- $ca := genCA (printf "%s-webhook-ca" .Release.Name  ) 36500 }}
+{{- $cert := genSignedCert $cn (list "127.0.0.1") (list $cn "localhost") 36500 $ca }}
+{{- $caCert := $ca.Cert | b64enc }}
+{{- $tlsCert := $cert.Cert | b64enc }}
+{{- $tlsKey := $cert.Key | b64enc }}
+{{- $_ := set . "CaCert" $caCert -}}
+{{- $_ := set . "TlsCert" $tlsCert -}}
+{{- $_ := set . "TlsKey" $tlsKey -}}
```

Comment:
this should be close the secret

Reply:
I have separated this file because somehow this global variables' declaration works only when this file is last when you arrange the `Helm` templates files alphabetically. I am not sure if it works when I include it in the same file as the secret.